internal/x11/wm/events.go

```diff
@@ -4,15 +4,15 @@ package wm
 
 import (
 	"fyne.io/fyne"
-	"github.com/BurntSushi/xgb/xproto"
```

Comment:
I don't know which IDE you are using, but I think normally the "local imports" come last, which is why it usually goes fyne -> github -> fynedesk.